### ISSUE TITLE
Failed to install - unsupported proxy error message

### ISSUE THREAD

**Issue by valentinitnelav** [valentinitnelav/plotbiomes#6]
When installing the `plotbiomes` package one can get something like below:

```r
devtools::install_github("valentinitnelav/plotbiomes")
## Error: Failed to install 'unknown package' from GitHub:
## Unsupported proxy 'https://proxy.companydomain:8080', libcurl is built without the HTTPS-proxy support.
```

Same happens with `remotes::install_github("valentinitnelav/plotbiomes")`.

Note that, `https://proxy.companydomain:8080/` is just an example here (can be whatever the case).

**Comment by valentinitnelav** [comment]
By the way, note that the error message doesn't catch the package's name. It gives `'unknown package'`. Check into that as well.

**Comment by valentinitnelav** [comment]
This was a problem regarding my user profile, environmental variables and admin group policies. The proxy issue got solved by the IT. This is not an R problem.
Curious though why the error message didn't catch the package name...
